src/main/java/io/logz/guice/jersey/JerseyServer.java

```diff
@@ -74,7 +78,23 @@ protected Injector getInjector() {
             }
         });
 
-        server.setHandler(webAppContext);
+        setHandler(server, webAppContext);
+    }
+
+    private void setHandler(HandlerWrapper handlerWrapper, Handler handlerToAdd) {
```

Comment:
Just to be clear (it took quite sometime to figure this all object hierarchy here): This method is static, and declaring it as such will help a bit to clarify.